ui.frontend/src/main/webpack/site/main.scss

```diff
@@ -1,6 +1,7 @@
-@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,600,700,800');
+@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800,900');
```

Comment:
Do we need all these weights?

Reply:
i am guessing auto merge added all again(i removed 500-medium and 900-black). We pretty much need 4 variants-- 300(lighter), 400(regular), 600(semibold) and 700(bold) as per screens provided.

Reply:
changes added in next commit